datalad/metadata/metadata.py

```diff
@@ -70,6 +76,21 @@
 from datalad.log import log_progress
 from datalad.core.local.status import get_paths_by_ds
 
+# Check availability of new-generation metadata
+try:
+    from datalad_metalad.dump import Dump
+    from datalad_metalad.exceptions import NoMetadataStoreFound
```

Comment:
this line is claimed to be not test covered -- so this "new" thing is not yet released or is there another reason for ImportError being raised?... oh well -- we do not submit coverage from extensions testing at all, filed https://github.com/datalad/datalad/issues/6780